virt/virt-2-4-release-notes.adoc

```diff
@@ -231,3 +231,28 @@ the nodes of any clusters where {VirtProductName} is deployed. The nodes might n
 drain if there are virtual machines running on top of them.
 
 ** The current solution is to xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance[put nodes into maintenance].
+
+* Importing a RHV VM fails if the target VM name exceeds 23 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
```

Comment:
The previous release note said 63 characters instead of 23. 
I see that both are mentioned in the bug. If the change to 23 chars is correct then the callout in the other PR I'm reviewing also needs to be changed.

Reply:
Good catch. It's 63